### ISSUE TITLE
Add Command Line Instructions Link for Merging Pull Requests

### ISSUE THREAD

**Issue by bagasme** [go-gitea/gitea#8270]
- Gitea version (or commit ref): 1.9.0
- Git version: 2.17.1
- Operating system: Ubuntu 18.04
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:
(not relevant)

## Description
In some cases when automatic merge PR cannot be done or whether merge button isn't desired, manual merging via command line can be performed instead.

Here are steps to perform manual merging (steps quoted from GitHub):
1. From your project's repository, fetch the changes from a PR, merge from `master`, and test:
```
git fetch origin
git checkout -b someone-patch-1-pr origin/someone-patch-1-pr
git merge master
```
2. Merge the changes back to `master` (`--no-ff`) and push:
```
git checkout master
git merge --no-ff someone-patch-1-pr
git push origin master
```
## Screenshots

This screenshot is from my `localhost` Gitea instance. Note that there isn't `view command line instructions` link beside merge button:
![image](https://user-images.githubusercontent.com/40219486/65478452-fa380b80-deb3-11e9-8ef4-1fc9f202d551.png)

Merge PR button on GitHub contains `view command line instructions` link. When the link is clicked, it opened full instructions, similar to the description above:
![image](https://user-images.githubusercontent.com/40219486/65478582-77638080-deb4-11e9-9497-f33ba979a2c8.png)

**Comment by bagasme** [comment]
The merging instructions above is similar to when hitting `Merge Pull Request` button regardless of merging method chosen in the button.

**Comment by MarkusAmshove** [comment]
This needs to take into consideration if the user is allowed to push into the target branch if it is protected

**Comment by na-Itms** [comment]
This seems to have been addressed as part of a9cc9c0, then improved in 2c04595 and 7a286e4. This issue now looks like it could be closed.

**Comment by delvh** [comment]
As per @na-Itms
